apps/epl/src/epl_tracer.erl

```diff
@@ -239,6 +270,10 @@ handle_info(timeout,
                                     Proplist,
                                     EtsTables),
 
+            {value, {timeline, Timelines}, PropsListWithoutFaultyTimelines } = lists:keytake(timeline, 1, Proplist1),
```

Comment:
What does `PropsListWithoutFaultyTimelines` mean? Is there a possibility we get some faulty timelines? Sorry if that's a silly question, I'm just curious.